cinema/serializers.py

```diff
@@ -59,6 +68,7 @@ class MovieSessionListSerializer(MovieSessionSerializer):
     cinema_hall_capacity = serializers.IntegerField(
         source="cinema_hall.capacity", read_only=True
     )
+    tickets_available = serializers.IntegerField(read_only=True)
```

Comment:
The `tickets_available` field is defined as read-only, but there is no logic provided to calculate or populate this field. Ensure that you implement a method or property in the `MovieSession` model to calculate the available tickets, or override the `to_representation` method to include this logic.